spec/cypress/integration/admin-create-administrator.js

```diff
@@ -0,0 +1,39 @@
+describe("Admin user creating another admin user", () => {
+  beforeEach(() => {
+    cy.login("admin");
+  });
+
+  it("should show a create administrator button", () => {
+    cy.visit("/admin/administrators");
+    cy.get("a.govuk-button").contains("Create a new administrator");
+  });
+
+  it("should create a new user", () => {
+    const fullName = "John Smith";
+    const email = "j.smith@example.com";
+
+    cy.visit("/admin/administrators");
+    cy.get("a.govuk-button").contains("Create a new administrator").click();
+
+    cy.get("h1").should("contain", "Create a new administrator user");
+    cy.get("input[name='user[full_name]']").type(fullName);
+    cy.get("input[name='user[email]']").type(email);
+    cy.get("input.govuk-button").contains("Continue").click();
```

Comment:
I would remove the `input` selector here and just have `.govuk-button` - it feels overly specific to me. The test ideally shouldn't break if it changes to a button or anchor element.